maat-court-data-api/src/main/java/gov/uk/courtdata/eform/service/EformStagingDAOImpl.java

```diff
@@ -0,0 +1,63 @@
+package gov.uk.courtdata.eform.service;
+
+import com.amazonaws.xray.spring.aop.XRayEnabled;
+import gov.uk.courtdata.eform.dto.EformStagingDTO;
+import gov.uk.courtdata.eform.mapper.EformStagingDTOMapper;
+import gov.uk.courtdata.eform.repository.EformStagingRepository;
+import gov.uk.courtdata.eform.repository.entity.EformsStagingEntity;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Propagation;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.Optional;
+
+/**
+ * The responsibility of this class is to provide data repository access without verification,
+ * verification is the responsibility of the calling class.
+ * e.g. verify that a given entity exists before attempting to delete it
+ */
+@Component
+@RequiredArgsConstructor
+@Slf4j
+@XRayEnabled
+public class EformStagingDAOImpl implements EformStagingDAO {
```

Comment:
Can we change the class name to EformService or a meaningful name without DAO to align with the MAAT API  naming convention?

Reply:
changed to EFormsStagingService